sdk/identity/identity/src/msal/flows.ts

```diff
@@ -18,6 +18,7 @@ export interface MsalFlowOptions {
   authorityHost?: string;
   authenticationRecord?: AuthenticationRecord;
   disableAutomaticAuthentication?: boolean;
+  allowPiiLogging?: boolean;
```

Comment:
where do we pass this on to the  MSAL client ?
Which part enables MSAL logging ?

Reply:
It’s a bit odd to track, but the MSAL client sends us whether a log line has PII or not through their log callbacks. We receive that parameter here: https://github.com/Azure/azure-sdk-for-js/pull/17321/files#diff-b4fea3e025941e1a4d4d02990dc8c9a31d77db91dec08c490f2cf818fee97111R94-R95 (it’s the one named “containsPii”). So, MSAL always sends us logs regardless of PII, we decide whether to log them or not.